src/selectors/getCurrentInputValue.js

```diff
@@ -0,0 +1,3 @@
+export default function getCurrentInputValue(state) {
```

Comment:
Let’s be more specific here since all selectors live in a single namespace, e.g. `getCurrentConsoleInputValue`.